installer/install_shimming_toolbox.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -e
+
+VENV_ID=1267b18e73341ad94da34474
+VENV=pst_venv_$VENV_ID
+ST_DIR=$HOME/shimming_toolbox
+PYTHON_DIR=python
+BIN_DIR=bin
+
+# Gets the shell rc file path based on the default shell.
+# @output: THE_RC and RC_FILE_PATH vars are modified
+function get_shell_rc_path() {
+  if [[ "$SHELL" == *"bash"* ]]; then
+    THE_RC="bash"
+    RC_FILE_PATH="$HOME/.bashrc"
+  elif [[ "$SHELL" == *"/sh"* ]]; then
+    THE_RC="bash"
+    RC_FILE_PATH="$HOME/.bashrc"
+  elif [[ "$SHELL" == *"zsh"* ]]; then
+    THE_RC="bash"
+    RC_FILE_PATH="$HOME/.zshrc"
+  elif [[ "$SHELL" == *"csh"* ]]; then
+    THE_RC="csh"
+    RC_FILE_PATH="$HOME/.cshrc"
+  else
+    find ~/.* -maxdepth 0 -type f
+    die "ERROR: Shell was not recognized: $SHELL"
+  fi
+}
+
+# Define sh files
+get_shell_rc_path
+
+# Update PATH variables based on Shell type
+DISPLAY_UPDATE_PATH="export PATH=\"$ST_DIR/$BIN_DIR:\$PATH\""
+
+# Installation text to insert in shell config file
+function edit_shellrc() {
+  # Write text common to all shells
+  (
+    echo
+    echo ""
+    echo "# SHIMMINGTOOLBOX (installed on $(date +%Y-%m-%d\ %H:%M:%S))"
+    echo "$DISPLAY_UPDATE_PATH"
```

Comment:
The reason I did this was because I wanted the bash scripts `install_plugin.sh` and `install_shimming_toolbox.sh` to be standalone, so you can just call one or the other if needed, which could be helpful for developing? I agree though it isn't very clean, and if you reinstall it keeps adding more lines 😅

Reply:
I made a fix here: https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox/pull/2/commits/4332505083389ccf174dafcf0e849372dd2a753a, so if the package has been previously installed, it won't keep adding lines to the `*.shrc` file.